<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GobangClass</class>
 <widget class="QWidget" name="GobangClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>五子棋—本地人人对战</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Gobang.qrc">
    <normaloff>:/Gobang/Image/Icon/图标.jpg</normaloff>:/Gobang/Image/Icon/图标.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="ChessBoard" name="checkerboard_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>840</width>
     <height>840</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Gobang/Image/Background/check.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="undo_button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>700</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Gobang/Image/Button/150.50.jpg);color: rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restart_button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>800</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Gobang/Image/Button/150.50.jpg);color: rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenu_button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>90</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(250, 250, 250);
background-image: url(:/Gobang/Image/Button/150.50.jpg);</string>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-image: url(:/Gobang/Image/Background/Gme1.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>190</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Gobang/Image/Button/150.50.jpg);
color: rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>保存棋局</string>
   </property>
  </widget>
  <zorder>background_label</zorder>
  <zorder>checkerboard_label</zorder>
  <zorder>undo_button</zorder>
  <zorder>restart_button</zorder>
  <zorder>mainMenu_button</zorder>
  <zorder>save_button</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ChessBoard</class>
   <extends>QLabel</extends>
   <header location="global">chessboard.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Gobang.qrc"/>
 </resources>
 <connections/>
</ui>
